JapaneseFilter/sections/specific.txt

```diff
@@ -40,6 +40,8 @@ hpplus.jp,tv-tokyo.co.jp,financial-field.com,spaia.jp,yomiuri.co.jp,fujinkoron.j
 !
 !
 pokemongo-kouryakumatome.net##.Article__content > div[align="center"] > br
+oshiete.goo.ne.jp##div[id^="gooad-long"]
```

Comment:
But we already have the necessary rule
![image](https://user-images.githubusercontent.com/8361299/165383874-2643cb1e-220d-412b-82c4-6d657d88f1a3.png)
Removed during optimization?

Reply:
I see this rule on optimized [filter](https://filters.adtidy.org/ios/filters/7_optimized.txt).